app/models/miq_worker/systemd_common.rb

```diff
@@ -7,11 +7,23 @@ def service_base_name
         "manageiq-#{minimal_class_name.underscore.tr("/", "_")}"
       end
 
+      def service_file
+        "#{service_base_name}@.service"
+      end
+
+      def target_file
+        "#{service_base_name}.target"
+      end
+
       def systemd_unit_dir
         Pathname.new("/lib/systemd/system")
       end
     end
 
+    def unit_name
+      "#{service_base_name}#{unit_instance}.service"
```

Comment:
Are the class method accessible from an instance method like this?  I can't remember if `class_methods` provides for that.

Reply:
Yeah it was private before, but the container_common side exposes container_image and container_name as public so it seemed symmetrical to expose this as well.